src/MapPanel.tsx

```diff
@@ -404,3 +415,6 @@ export class MapPanel extends Component<MapPanelProps> {
     return elem;
   }
 }
+
+const mapPanelWithTheme = withTheme(MapPanel);
```

Comment:
I think the concept here is that we are wrapping in some thingy that has theme data associated with it for the purposes of comparing the color data in a testing context? Maybe do this in a separate testing-oriented file that imports MapPanel directly instead of the high-order component and fiddling with the exports